include/sys/uio_impl.h

```diff
@@ -42,8 +42,40 @@
 #include <sys/uio.h>
 
 extern int zfs_uiomove(void *, size_t, zfs_uio_rw_t, zfs_uio_t *);
-extern int zfs_uio_prefaultpages(ssize_t, zfs_uio_t *);
 extern int zfs_uiocopy(void *, size_t, zfs_uio_rw_t, zfs_uio_t *, size_t *);
 extern void zfs_uioskip(zfs_uio_t *, size_t);
 
+static inline void
+zfs_uio_setoffset(zfs_uio_t *uio, offset_t off)
+{
+	zfs_uio_offset(uio) = off;
+}
```

Comment:
Let's move this in to the platform specific headers so we don't need to use the `zfs_uio_offset()` wrapper here when assigning a value.

Reply:
This has been moved back to the OS specific uio.h.